jenkins/development.Dockerfile.j2

```diff
@@ -110,6 +109,7 @@ RUN \
         `# file and this list should be regenerated.` \
         `#` \
         cronie \
+        libpq-devel \
```

Comment:
Something else pulls it in.

Neither file lists the full dependency tree, since each RPM sub-dependency contributes as well.

Reply:
We're including `libpq-devel` here because we know that the pip-install of `psycopg2` will need it, so it seems like we ought to be (explicitly) listing `gcc` here as well, since we know that the pip-install will need that, too.  The fact that something else (implicitly) pulls it in for us is fine...as long as we keep pulling that thing in and as long as its behavior doesn't change.  I mean, we can go with the "it seems to work" definition of success, but it seems like it would be better for us to state our requirements (to the extent that we know them...) and let `dnf` decide whether they are already satisfied.